src/20250203_Multi_UpdateUSDSGHOBorrowRate/UpdateUSDSGHOBorrowRate.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Update USDS & GHO Borrow Rate"
+author: "Aave-chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-update-usds-gho-borrow-rate/20892"
+---
+
+## Simple Summary
+
+The Sky Savings Rate (SSR) is expected to be lowered from 12.50% to 9.50% on February 10th. This publication proposes updating the USDS Borrow Rate and also, reducing the GHO Borrow Rate on the Prime instance.
+This update will be conducted via a Direct-to-AIP process.
+
+## Motivation
+
+In response to the latest proposal to reduce the SSR to [9.50%](https://forum.sky.money/t/feb-6-2025-stability-scope-parameter-changes-21/25906), this publication proposes reducing the Base on Core and Slope1 on Prime by 3.00% respectively.
+
+The chart below shows the USDC and USDT deposit rates on Aave relative to the SSR over the last 90 days. More recently, the SSR has materially outperformed USDC and USDT deposit yield on Aave due to the borrow rate for USDC and USDT continually declining since mid-december.
+
+The GHO Borrow Rate on Core is currently configured at 9.00%, representing a cost of capital of 9.42% for non stkAAVE holders. 9.42% compares favourably to the SUSDS Deposit Yield (Native + Incentives) of 13.42% + SPK airdrop rewards.
+
+When the SSR is reduced to 9.50%, the USDS collateral and GHO debt free carry trade remains profitable with compressed margins until such time as the GHO Borrow Rate is lowered. Notably, over the last 30 days, wallets holding USDS deposits on Core have borrowed +1,108,212 units of GHO representing a small portion of newly borrowed GHO.
+
+With the GHO Steward role in place on Core, the GHO Borrow Rate can be reduced with short notice and therefore, this publication focuses on adjusting the Prime instance of Aave where the GHO Steward role is not yet extended.
+
+With the USDS Borrow Rate, Slope1 parameter, being reduced by 3.00% this publication proposes reducing the GHO Borrow Rate, Base parameter by 1.50%. Upon implementing both Borrow Rate updates, the GHO Borrow Rate at the Uoptimal will be 0.50% less than USDS. A slightly lower GHO Borrow Rate provides an economic incentive to hold GHO debt relative to USDS.
+
+## Specification
+
+**Prime Instance**
+
+USDS
+
+| Detail | Current | Proposed | Change |
+| ------ | ------- | -------- | ------ |
+| Base   | 0.75%   | 0.75%    | 0.00%  |
+| Slope1 | 11.75%  | 8.75%    | -3.00% |
```

Comment:
In diffs current is shown at 10.75% . same on the contract https://etherscan.io/address/0x8958b1C39269167527821f8c276Ef7504883f2fa#readContract 
So table should be changed acordingly